Documentation/Extensions/SceneTransitionService/SceneTransitionServiceOverview.md

```diff
@@ -0,0 +1,187 @@
+        );
+}
+
+private async Task ResetScene() 
+{
+    // Go through all enemies in the current scene and move them back to starting positions   
+}
+```
+
+Or you may want to load a scene without using the SceneSystem service:
+```
+private async void TransitionToScene()
+{
+    ISceneTransitionService transition = MixedRealityToolkit.Instance.GetService<ISceneTransitionService>();
+
+    // Fades out
+    // Loads scene using Unity's scene manager
+    // Fades back in
+    await transition.DoSceneTransition(
+            () => LoadScene("TestScene1")
+        );
+}
+
+private async Task LoadScene(string sceneName)
+{
+    AsyncOperation asyncOp = SceneManager.LoadSceneAsync(sceneName, LoadSceneMode.Additive);
+    while (!asyncOp.isDone)
+    {
+        await Task.Yield();
+    }
+}
+```
+
+## Using Multiple Tasks
+You can also supply multiple tasks, which will be executed in order:
+```
+private async void TransitionToScene()
+{
+    IMixedRealitySceneSystem sceneSystem = MixedRealityToolkit.Instance.GetService<IMixedRealitySceneSystem>();
+    ISceneTransitionService transition = MixedRealityToolkit.Instance.GetService<ISceneTransitionService>();
+
+    // Fades out
+    // Sets time scale to 0
+    // Fades out audio to 0
+    // Loads TestScene1
+    // Fades in audio to 1
+    // Sets time scale to 1
+    // Fades back in
+    await transition.DoSceneTransition(
+            () => SetTimescale(0f),
+            () => FadeAudio(0f, 1f),
+            () => sceneSystem.LoadContent("TestScene1"),
+            () => FadeAudio(1f, 1f),
+            () => SetTimescale(1f)
+        );
+}
+
+private async Task SetTimescale(float targetTime)
+{
+    Time.timeScale = targetTime;
+    await Task.Yield();
+}
+
+private async Task FadeAudio(float targetVolume, float duration)
+{
+    float startTime = Time.realtimeSinceStartup;
+    float startVolume = AudioListener.volume;
+    while (Time.realtimeSinceStartup < startTime + duration)
+    {
+        AudioListener.volume = Mathf.Lerp(startVolume, targetVolume, Time.realtimeSinceStartup - startTime / duration);
+        await Task.Yield();
+       }
+       AudioListener.volume = targetVolume;
+}
+```
+
+## Using the Progress Indicator
+
+A progress indicator is anything that implements the `IProgressIndicator` interface. This can take the form of a splash screen, a 3D tagalong loading indicator, or anything else that provides feedback about transition progress.
+
+If `UseDefaultProgressIndicator` is checked in the SceneTransitionService profile, a progress indicator will be instantiated when a transition begins. For the duration of the transition this indicator's `Progress` and `Message` properties can be accessed via that service's `SetProgressValue` and `SetProgressMessage` methods.
+
+Alternatively, when calling `DoSceneTransition` you can supply your own progress indicator via the optional `progressIndicator` argument. This will override the default progress indicator.
+
```

Comment:
Nit: It might be worth moving this to AFTER the code example, because it isn't highlighted/listed in the code example (I was looking for it, didn't find it, and then realized that the code example is more talking about the earlier stuff)

Reply:
Sorry I'm not sure what bits 'this' and 'the code example' and 'earlier stuff' are referring to, exactly.